From b3a7c1384cd3f43266b32a1174f1f8e624fd2719 Mon Sep 17 00:00:00 2001
Message-ID: <b3a7c1384cd3f43266b32a1174f1f8e624fd2719.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:52:59 +0000
Subject: [PATCH v3 57/66] gnu: go-github-com-xeipuuv-gojsonschema: Adjust
 inputs.

* gnu/packages/golang-web.scm (go-github-com-xeipuuv-gojsonschema):
[propagated-inputs]: Move go-github-com-stretchr-testify from here ...
[native-inputs]: ... to here.

Change-Id: Id865d5821060cd847aeb9ccfc7b7a477e12f7b7b
---
 gnu/packages/golang-web.scm | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index c97dfa7757..346ca87cc7 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -1232,10 +1232,11 @@ (define-public go-github-com-xeipuuv-gojsonschema
                     (string-append "// " all))
                    (("\\{\"phase\": \"invalid definition" all)
                     (string-append "// " all)))))))))
+      (native-inputs
+       (list go-github-com-stretchr-testify))
       (propagated-inputs
        (list go-github-com-xeipuuv-gojsonreference
-             go-github-com-xeipuuv-gojsonpointer
-             go-github-com-stretchr-testify))
+             go-github-com-xeipuuv-gojsonpointer))
       (home-page "https://github.com/xeipuuv/gojsonschema")
       (synopsis "Implementation of JSON Schema for Go")
       (description
-- 
2.41.0

